### PR TITLE
Balance Auto

### PR DESCRIPTION
**Pull request by Speedy6451**
Confirmed to score points at [@53ee559](https://github.com/Team-2502/RobotCode2023/commit/53ee559506cf476c6cf58c3a2a011fae138886d3)

based on [week zero](https://github.com/Team-2502/RobotCode2023/tree/kcw0) branch

### CODE REVIEW

File: src/main/java/com/team2502/robot2023/RobotContainer.java [hunk 1]
```diff
@@ -64,19 +59,22 @@ private void configureButtonBindings() {
...
-        RunIntake.whileTrue(new RunIntakeCommand(INTAKE, CONVEYOR, 0.5, 0.5, 0.6));
+        RunIntake.whileTrue(new RunIntakeCommand(INTAKE, CONVEYOR, 0.5, 0.55, 0.4));
 
         JoystickButton RunIntakeBack = new JoystickButton(JOYSTICK_DRIVE_LEFT, Constants.OI.RUN_INTAKE_BACK);
-        RunIntakeBack.whileTrue(new RunIntakeCommand(INTAKE, CONVEYOR, -0.5, -0.5, -0.6));
+        RunIntakeBack.whileTrue(new RunIntakeCommand(INTAKE, CONVEYOR, -0.5, -0.55, -0.4));
 
         JoystickButton ElevatorBot = new JoystickButton(JOYSTICK_OPERATOR, Constants.OI.ELEVATOR_BOT);
-        ElevatorBot.onTrue(new RunElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.BOTTOM));
+        ElevatorBot.onTrue(new SetElevatorCommand(ELEVATOR, Constants.Subsystems.Elevator.ElevatorPosition.BOTTOM))
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/RobotContainer.java:68]: ?

File: src/main/java/com/team2502/robot2023/subsystems/DrivetrainSubsystem.java [hunk 2]
```diff
@@ -379,6 +389,17 @@ public double getHeading() {
         return navX.getAngle();
     }
 
+    public void resetPitch() {
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/subsystems/DrivetrainSubsystem.java]: Recommend renaming to "resetRoll" to reduce confusion in naming

File: src/main/java/com/team2502/robot2023/subsystems/IntakeSubsystem.java [hunk 3]
```diff
@@ -32,9 +32,14 @@ public IntakeSubsystem() {
         rightMotor.setSmartCurrentLimit(39);
         leftMotor.setSmartCurrentLimit(39);
 
+        leftLiftMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
+        rightLiftMotor.setIdleMode(CANSparkMax.IdleMode.kBrake);
```
Comment (isaacmyers11) [inline src/main/java/com/team2502/robot2023/subsystems/IntakeSubsystem.java:36]: Why have intake brake when done?

**Review by isaacmyers11 — APPROVED** [review]

File: src/main/java/com/team2502/robot2023/subsystems/DrivetrainSubsystem.java (hunk 2 above)
Comment (Speedy6451) [reply src/main/java/com/team2502/robot2023/subsystems/DrivetrainSubsystem.java]: resolved by [15b60](https://github.com/Team-2502/RobotCode2023/pull/9/commits/15b60829f4d6390c5c0c9d459b2de9871e215676)

File: src/main/java/com/team2502/robot2023/subsystems/IntakeSubsystem.java (hunk 3 above)
Comment (Speedy6451) [reply src/main/java/com/team2502/robot2023/subsystems/IntakeSubsystem.java:36]: resolved by [be680](https://github.com/Team-2502/RobotCode2023/pull/9/commits/be68063018a19a50d1813a2938754e698cdb4a1a)

**Pull request merged**